services/ext/api.go

```diff
@@ -797,6 +797,7 @@ func (api *PublicAPI) DeleteMessagesByChatID(id string) error {
 	return api.service.messenger.DeleteMessagesByChatID(id)
 }
 
+// Deprecated: Use MarkMessagesRead instead
```

Comment:
I believe you should use the deprecated pragma instead, because a comment is not enough to help future developers

Example : 

```
func (api *PublicAPI) MarkMessagesSeen(chatID string, ids []string) (*MarkMessageSeenResponse, error) {.deprecated: "use MarkMessagesRead instead".}
```

Reply:
This is part of go, and it seems to me that `.deprecated` pragma is from nim.  In status-go project `// Deprecated:` comment its a common way to do it

Reply:
oh my bad, I didn't read well here 😄